scene/resources/primitive_meshes.h

```diff
@@ -262,6 +262,15 @@ class PlaneMesh : public PrimitiveMesh {
 
 VARIANT_ENUM_CAST(PlaneMesh::Orientation)
 
+class QuadMesh : public PlaneMesh {
```

Comment:
Could use a short comment describing it like other classes - just for visual grepping in that header to easily notice where classes start and end.